pkg/cloudprovider/providers/rackspace/rackspace.go

```diff
@@ -558,14 +558,18 @@ func (os *Rackspace) GetZone() (cloudprovider.Zone, error) {
 // This is particularly useful in external cloud providers where the kubelet
 // does not initialize node data.
 func (os *Rackspace) GetZoneByProviderID(providerID string) (cloudprovider.Zone, error) {
-	return cloudprovider.Zone{}, errors.New("GetZoneByProviderID not implemented")
+	glog.V(1).Infof("Current zone is %v", os.region)
```

Comment:
Are these log statements necessary? I can see how they are useful for initial debugging, but this code is pretty straightforward so I don't see why they should be checked in.

Reply:
Right, they seems unnecessary, remove them like gce and aws.